modules/structs/pull_review.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2020 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package structs
+
+import (
+	"time"
+)
+
+// PullRequestReview represents a pull request review
+type PullRequestReview struct {
+	ID       int64  `json:"id"`
+	PRURL    string `json:"pull_request_url"`
+	Reviewer *User  `json:"user"`
+	Body     string `json:"body"`
+	CommitID string `json:"commit_id"`
+	Type     string `json:"type"`
+
```

Comment:
can you add state info too?
```json
"state": "APPROVED",
```

Reply:
o it still a draft :D - the start looks good  :+1:

Reply:
I thought about naming that `state`, but it was so misleading, that I've gone with `type`.
```
"state":"COMMENT"
```
just looks strange.

But i can rename this if wanted.

Reply:
fair point ... just thought did a comparsion with the github api ...
and beside COMMENT, PENDING, APPROVED, REJECT could interpreted as state ...

Reply:
> But i can rename this if wanted.

please :)

Reply:
Maybe add field `submitted_at`?

Reply:
`stale` and `official` can also be interesting fields (GH doesn't have them)